backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/optimizations/LocalEscapeAnalysis.kt

```diff
@@ -74,6 +74,11 @@ internal object LocalEscapeAnalysis {
                         } ?: node.callee.irFunction?.isExternal?.let {
                             !(it || node.callee.irFunction.isBuiltInOperator)
```

Comment:
`isBuiltInOperator` is not needed anymore, isn't it? Since we're replacing call to `get` with `ArrayGet`.

Reply:
Do any other builtinOperator exist? I amn'tt sure and as far as there is `isBuiltinOperator` https://github.com/JetBrains/kotlin-native/blob/d6b20598e82be6a35c003fcfacc7dbda38a897be/backend.native/compiler/ir/backend.native/src/org/jetbains/kotlin/backend/konan/optimizations/DataFlowIR.kt#L600 I left it.